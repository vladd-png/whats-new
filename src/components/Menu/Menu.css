nav {
  color: white;
  grid-column: 1;
  grid-row: 1 / span 2;
}

.newsType {
  /* background-color: #141527; */
  height: 60px;
  width: 100%;
  text-align: left;
  border-radius: 0px 50px 50px 0px;
  margin-top: 30px;
}

.newsType:hover {
  background-color: #FEAE01;
}

.newsTitle {
  color: #FEAE01;
}

h1 {
  font-size: 40px;
  margin: 10px;
}
