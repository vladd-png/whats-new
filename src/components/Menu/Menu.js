import React, { Component } from 'react';
import './Menu.css';
import location from '../../loaction.png';
import technology from '../../lightbulb.png';
import entertainment from '../../tickets.png';
import science from '../../rocket.png';
import health from '../../health.png';


class Menu extends Component {
  constructor() {
    super();
    this.state = {
      newsType: 'local'
    }
  }
  handleChange = (newsString) => {
    this.setState({newsType: newsString});
  }
  render() {
    return (
      <nav>
        <h1>What's <span className='newsTitle'>New ?</span></h1>
        <ul>
          <li className='newsType' onClick={() => this.handleChange('local')}><img className='imgIcon' src={location} alt='news icon' />Local News</li>
          <li className='newsType' onClick={() => this.handleChange('technology')}><img className='imgIcon' src={technology} alt='news icon' />Technology</li>
          <li className='newsType' onClick={() => this.handleChange('entertainment')}><img className='imgIcon' src={entertainment} alt='news icon' />Entertainment</li>
          <li className='newsType' onClick={() => this.handleChange('science')}><img className='imgIcon' src={science} alt='news icon' />Science</li>
          <li className='newsType' onClick={() => this.handleChange('health')}><img className='imgIcon' src={health} alt='news icon'/>Health</li>
        </ul>
      </nav>
    )
  }
}

export default Menu;
