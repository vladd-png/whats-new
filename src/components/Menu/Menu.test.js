import React from 'react';
import { shallow } from 'enzyme';
import Menu from './Menu';
describe('Menu', () => {
  it('', () => {
    const wrapper = shallow(<Menu
      handleChange={jest.fn()}
    />);
    expect(wrapper).toMatchSnapshot();
  })
})
