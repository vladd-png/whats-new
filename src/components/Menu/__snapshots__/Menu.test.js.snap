// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu  1`] = `
<nav>
  <h1>
    What's 
    <span
      className="newsTitle"
    >
      New ?
    </span>
  </h1>
  <ul>
    <li
      className="newsType"
      onClick={[Function]}
    >
      <img
        alt="news icon"
        className="imgIcon"
        src="loaction.png"
      />
      Local News
    </li>
    <li
      className="newsType"
      onClick={[Function]}
    >
      <img
        alt="news icon"
        className="imgIcon"
        src="lightbulb.png"
      />
      Technology
    </li>
    <li
      className="newsType"
      onClick={[Function]}
    >
      <img
        alt="news icon"
        className="imgIcon"
        src="tickets.png"
      />
      Entertainment
    </li>
    <li
      className="newsType"
      onClick={[Function]}
    >
      <img
        alt="news icon"
        className="imgIcon"
        src="rocket.png"
      />
      Science
    </li>
    <li
      className="newsType"
      onClick={[Function]}
    >
      <img
        alt="news icon"
        className="imgIcon"
        src="health.png"
      />
      Health
    </li>
  </ul>
</nav>
`;
